lib/typescript/assets/images/icons/template-alt.svg

```diff
@@ -1 +1,6 @@
-<svg height="34" width="31" xmlns="http://www.w3.org/2000/svg"><path d="M30.5 25.333v5a3.333 3.333 0 0 1-3.333 3.334H3.833A3.333 3.333 0 0 1 .5 30.333v-5zM28.833 27H2.167v3.333c0 .855.643 1.56 1.472 1.656l.194.011h23.334c.854 0 1.559-.643 1.655-1.472l.011-.195zM27.167.333A3.333 3.333 0 0 1 30.5 3.667V22H.5V3.667A3.333 3.333 0 0 1 3.833.333zm0 1.667H3.833c-.854 0-1.559.643-1.655 1.472l-.011.195v16.666h26.666V3.667c0-.855-.643-1.56-1.472-1.656z" fill="#212428" fill-rule="evenodd"/></svg>
\ No newline at end of file
+<?xml version="1.0" encoding="UTF-8"?>
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="20pt" height="21pt" viewBox="0 0 20 21" version="1.1">
+<g id="surface1">
+<path style=" stroke:none;fill-rule:evenodd;fill:rgb(12.941176%,14.117647%,15.686275%);fill-opacity:1;" d="M 19.675781 15.648438 L 19.675781 18.734375 C 19.679688 19.871094 18.714844 20.792969 17.527344 20.792969 L 2.472656 20.792969 C 1.285156 20.792969 0.320312 19.871094 0.324219 18.734375 L 0.324219 15.648438 Z M 18.601562 16.675781 L 1.398438 16.675781 L 1.398438 18.734375 C 1.398438 19.261719 1.8125 19.699219 2.347656 19.757812 L 2.472656 19.765625 L 17.527344 19.765625 C 18.078125 19.765625 18.53125 19.367188 18.59375 18.855469 L 18.601562 18.734375 Z M 17.527344 0.207031 C 18.714844 0.207031 19.679688 1.128906 19.675781 2.265625 L 19.675781 13.589844 L 0.324219 13.589844 L 0.324219 2.265625 C 0.320312 1.128906 1.285156 0.207031 2.472656 0.207031 Z M 17.527344 1.234375 L 2.472656 1.234375 C 1.921875 1.234375 1.46875 1.632812 1.40625 2.144531 L 1.398438 2.265625 L 1.398438 12.558594 L 18.601562 12.558594 L 18.601562 2.265625 C 18.601562 1.738281 18.1875 1.300781 17.652344 1.242188 Z M 17.527344 1.234375 "/>
+</g>
+</svg>
```

Comment:
We (I mean @chrismatix) need to automate this but can you please run https://github.com/svg/svgo on this? It'll strip the svg to the min needed